components/organisms/o-shipping.vue

```diff
@@ -0,0 +1,320 @@
+<template>
+  <div class="o-shipping">
+    <SfHeading
+      :title="`2. ${$t('Shipping')}`"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <div class="form">
+      <div
+        v-if="currentUser && hasShippingDetails()"
+        class="form__element form__group"
+      >
+        <SfCheckbox
+          v-model="shipToMyAddress"
+          class="form__checkbox"
+          name="shipToMyAddress"
+          :label="$t('Ship to my default address')"
+        />
+      </div>
+      <SfInput
+        v-model.trim="shipping.firstName"
+        class="form__element form__element--half"
+        name="first-name"
+        :label="$t('First name *')"
+        :valid="!$v.shipping.firstName.$error"
+        :error-message="
+          !$v.shipping.firstName.required
```

Comment:
We don't need to check second condition and this can be typed simpler: `$v.fieldName.requred ? $t('Field is required') : $t('Other message')`

Reply:
Done.